### ISSUE TITLE
Bred Dino with part multiplier

### ISSUE THREAD

**Issue by pfar1** [cadon/ARKStatsExtractor#204]
Hi,

I raised 2x rexes during the recent event after the AWS rollback, WC raised the maturation multiplier.

The rexes were raised at the back end of this and the event finished when the dinos were only half raised.

Now they are done, but the extractor cant extract the stats.

HP: 13569.4
ST: 1764.0
OX: 705.0
FD: 13958.8
WE: 876.5
DM: 440.9
SP: 108.2
TP: 27641.2

Imprint: 41%
Level: 259

I have tried playing with the babymaturespeedmultiplier.  I seem to be able to get the imprinting to come out at 41% if i set it to 2.373, but still wont extract properly.

Any help appreciated.

Thanks
Paul

**Comment by cadon** [comment]
Hi, afaik the maturation-multiplier has to be set to the value it was on the server when the babies hatched. The imprinting-value ingame is always only a rounded value, so a value of 41% ingame can represent any value in the range of 40.5 % to 41.499999 %.

**Comment by arkin-** [comment]
As a tribe, we are also having this issue, going to try edit this setting for dinosaurs bred during the events.

**Comment by arkin-** [comment]
Ok to further explain this issue, I'm unable to enter values with a 38% imprint on Ptera, it seems half was done at normal rates and the other half was done at 1.5x maturation rates. Even with various stats locked out to zero, and various amendments I cannot set 38% imprint.

**Comment by cadon** [comment]
The possible percentages are dependant on the BabyCuddleIntervallMultiplier and the BabyMaturationSpeedMultiplier. If one or both of them don't match the numbers the game uses for that creature, the extraction will most probably fail. Did you try to increase the maturation multiplier to 1.5? If you're on an official server, I'm not sure if the cuddle multiplier is affected by the evolution-events as well. It could be, that it has to be set to the reciprocal value of the maturation-speed (i.e. 0.6666), though I'm not sure about that last one.

**Comment by pfar1** [comment]
I am on official, i tried 1.5 on the maturation no luck, i seem to get better around 3... i tried tweaking the cuddle aswell, but to be honest i dont know enough about the multipliers and am literally just increasing/decreasing them until more stuff goes green ;)... 

if someone figures it out, please let me know... otherwise i will just breed these rexes manually.

**Comment by DarthMitsu** [comment]
I had the same problem, i just could enter 0 or 100%, everything between was set back to 0. After reading this thread I checked the server settings again and saw that i haven't adjusted the BabyCuddleIntervallMultiplier. After I've set it right everything works great with the not fully imprinted dinos!
Thanks for the solution and for this really handy program, makes breeding and comparing my dinos so much easier!
